config/jobs/kubernetes/kops/OWNERS

```diff
@@ -15,5 +15,3 @@ approvers:
 reviewers:
   - joshbranham
   - robinpercy
-labels:
-- sig/cluster-lifecycle
```

Comment:
I'm wondering if we should keep these. Even if we make kops its own testgrid dashboard, the project is still owned by sig-cluster-lifecycle

Reply:
We should add some labels I guess, but not sure if we want to keep this or add a new one more explicit.

Reply:
Labels are back :)